rumqttc/src/mqttbytes/mod.rs

```diff
@@ -76,7 +76,7 @@ pub enum QoS {
 
 /// Packet type from a byte
 ///
-/// ```ignore
+/// ```text
```

Comment:
This ensures that a doc test doesn't get triggered, the contents aren't testable code, they are diagrams